include/deal.II/base/symmetric_tensor.h

```diff
@@ -406,6 +411,11 @@ namespace internal
        */
       reference operator [] (const unsigned int);
 
+      /**
+       * Index operator.
+       */
+      reference operator [] (const unsigned int) const;
```

Comment:
@bangerth Should this operator always return a constant reference (i.e. an `Accessor` with `constness=true`)? Or is this fine as is?